SoloProject/scraper/newspapers/UK/daily-mail.js

```diff
@@ -0,0 +1,24 @@
+const got = require('got');
```

Comment:
you should modularize and try not to repeat yourself when possible ( sometimes you have to do it ) you could have created a function to scrape newspapers by parameters trying to extract the code they have in common so you dont have to create one file for each newspaper